homeassistant/components/discovergy/translations/de.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Translations are handled in Lokalise https://developers.home-assistant.io/docs/translations